scripts/xtalk_filter_run.py

```diff
@@ -11,7 +11,10 @@
 parser = xtalk_filter.xtalk_filter_argparser(mode='clean')
 
 a = parser.parse_args()
-
+# allow none string to be passed through to a.calfile
+if a.calfile is not None:
```

Comment:
Check if it's a string rather than if it's not None. It's also OK to do both ifs in one line.

Reply:
Made it one line.